build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -49,6 +50,41 @@ public static BuildTargetResult GenerateVersionBadge(BuildTargetContext c)
         }
         
         [Target]
+        [BuildPlatforms(BuildPlatform.Windows)]
+        public static BuildTargetResult CopyNativeSymbols(BuildTargetContext c)
+        {
+            var nativeSymbolsRoot = Path.Combine(Dirs.Output, "obj", "nativeSymbols");
+            var nativeSymbolsZipFileDir = Path.Combine(Dirs.Intermediate, "nativeSymbols");
+
+            Utils.DeleteDirectory(nativeSymbolsRoot);
+            Directory.CreateDirectory(nativeSymbolsRoot);
+
+            Utils.DeleteDirectory(nativeSymbolsZipFileDir);
+            Directory.CreateDirectory(nativeSymbolsZipFileDir);
+
+            foreach (var file in Directory.GetFiles(Dirs.CorehostLatest, "*.pdb", SearchOption.TopDirectoryOnly))
```

Comment:
@schellap The host binaries getting pushed to azure are consumed in the cli. It is the locked version of the corehost, and if it is latest that is a bug.